learning/tour-of-beam/frontend/lib/constants/font_weights.dart

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:flutter/cupertino.dart';
```

Comment:
I like including the minimal possible libraries. In this case it is `flutter/painting.dart`. But narrowing it down can be seen as a waste of time because unused things are not compiled anyway.

If you don't want to narrow down too much, it's better to go with `widgets.dart` or `material.dart`, because `cupertino` catches attention and makes one double-check if we have Apple styles anywhere.